etc/tune.c

```diff
@@ -64,19 +63,17 @@ static void s_raninit(uint64_t seed)
    source of the OS for its purpose. That is too expensive, too slow and
    most important for a benchmark: it is not repeatable.
 */
-static int s_mp_random(mp_int *a, int limbs)
+static int s_ranbuf(void *p, size_t n)
 {
-   int e = MP_OKAY;
-   if ((e = mp_grow(a, limbs + 1)) != MP_OKAY) {
-      goto LTM_ERR;
+   char *q = (char *)p;
+   while (n > 0) {
+      uint64_t x = s_ranval();
+      for (int i = 0; i < 8 && n > 0; ++i, --n) {
```

Comment:
Has c89/c90 support been officially dropped in LTM now? Not that I would shed a single tear, just asking.

Reply:
Haha always trying to sneak it in. We should probably set std=c89 to enforce it.